blog/poc3.md

```diff
@@ -1,120 +1,64 @@
 
 
-Swarm reaches a significant milestone as the next generation of its proof of concept releases (POC3) is now deployed on the public Testnet. The foundation is running a 50-node strong cluster (with a public gateway on https://swarm-gateways.net) hopefully growing as our collaborators switch on their permanent nodes and clusters.
-The POC3 code is now merged into the official [go-ethereum repository's master branch](https://github.com/ethereum/go-ethereum).
-It has been a year and a half since the first release of the POC2 series was deployed and [the Swarm project launched its public alpha network](https://blog.ethereum.org/2016/12/15/swarm-alpha-public-pilot-basics-swarm/).
-Two [Swarm summits](https://swarm-gateways.net/bzz:/swarm-summit.eth), two [orange papers](http://swarm-guide.readthedocs.io/en/latest/resources.html#orange-papers) and [forty thousand lines of code](https://github.com/ethereum/go-ethereum/pull/17041) later, it is time to give an update.
+The Swarm Team is pleased to announce the immediate release of Swarm 0.3, the third proof-of-concept release (POC3) of the Ethereum Swarm client.
```

Comment:
I wonder if we should use "client" for a p2p binary. "Node", "Binary" are other options, possibly there are better ones.

Reply:
Swarm client v0.3 ?